test/e2e/index.js

```diff
@@ -47,14 +48,14 @@ describe("Application launch", function () { // eslint-disable-line func-names
         });
 
         it("should can't proceed without accept eula", async () => {
-            await app.client.click("#agreement-submit");
+            await app.client.click("#submitbutton");
             const licenseExists = await app.client.isExisting(".license__text");
             assert.equal(licenseExists, true, "license should exists");
         });
 
         it("should proceed with eula", async () => {
             await app.client.click(".js-agreement");
-            await app.client.click("#agreement-submit");
+            await app.client.click("#submitbutton");
             await utils.sleep(3000);
```

Comment:
Use named consts from config instead of 3000

Reply:
Fixed